### PR TITLE
appdome-build-2secure-android 1.0.0

### PR DESCRIPTION
**Pull request by elish266**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/appdome-build-2secure-android/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    is_required: true
+    title: Signing Method
+    value_options:
+    - On-Appdome
+    - Private-Signing
+    - Auto-Dev-Signing
+  sign_method: On-Appdome
+- app_signing_cert: null
+  opts:
+    description: Google Play signing SHA-1/SHA256 fingerprint (will be used if provided)
```
Comment (koral--) [inline steps/appdome-build-2secure-android/1.0.0/step.yml:46]:
```suggestion
    description: Google Play signing SHA-1/SHA-256 fingerprint (will be used if provided)
```

**Comment by elish266** [comment]
I made the

**Comment by elish266** [comment]
Fixing the code according to your comments.
Will open a new pull request with a new tag.

**Comment by elish266** [comment]
Hello,

I have made all fixes you requested.
However my merging is still blocked.
Please advise.
Thanks,
Eli


From: Karol Wrótniak ***@***.***>
Reply-To: bitrise-io/bitrise-steplib ***@***.***>
Date: Thursday, 30 March 2023 at 17:00
To: bitrise-io/bitrise-steplib ***@***.***>
Cc: Eli ***@***.***>, Author ***@***.***>
Subject: Re: [bitrise-io/bitrise-steplib] appdome-build-2secure-android 1.0.0 (PR #3787)


@koral-- commented on this pull request.

________________________________

In steps/appdome-build-2secure-android/1.0.0/step.yml<https://github.com/bitrise-io/bitrise-steplib/pull/3787#discussion_r1153310876>:

> +- opts:

+    is_required: false

+    title: Team ID

+  team_id: null

+- opts:

+    description: App signing method

+    is_required: true

+    title: Signing Method

+    value_options:

+    - On-Appdome

+    - Private-Signing

+    - Auto-Dev-Signing

+  sign_method: On-Appdome

+- app_signing_cert: null

+  opts:

+    description: Google Play signing SHA-1/SHA256 fingerprint (will be used if provided)
⬇️ Suggested change

-    description: Google Play signing SHA-1/SHA256 fingerprint (will be used if provided)

+    description: Google Play signing SHA-1/SHA-256 fingerprint (will be used if provided)

**Pull request closed without merging**